Tunnel Client/minecraft/README.md

```diff
@@ -0,0 +1,74 @@
+# Super Protocol — Minecraft
+
+## Description
+This is the Minecraft server and client that can be run on SuperProtocol.
+
+Key features:
+- Server and client in one assembly
+- Server — [flying-squid](https://github.com/PrismarineJS/flying-squid)
+- Client — [prismarine-web-client](https://github.com/PrismarineJS/prismarine-web-client)
+
+The client includes a web-server that connects to the MS Server and provides the UI to the user in a web browser.
+
+## Local run
+Prerequisites:
+- Node.js (>18)
+- Yarn (v.1)
+
+
+To build and run in dev mode, rename the *.env.example* file to *.env* and fill in the appropriate parameters.
+
+Next, run the commands:
+```shell
+cd minecraft
+yarn dependencies
+yarn build:all
+yarn dev
+```
+
+## docker-compose
+Generate self-signed key and ssl-certificate for localhost:
+```shell
+openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.crt -sha256 -days 365 -nodes -subj "/CN=localhost"
+```
```

Comment:
Here also needed instruction for preparing .env file.
```
cp .env.example .env
```
There is alse needed commands to add cert and key to .env file after as TLS_* variables as single line.
Save single line to variable is possible by commnad 
`CONTENT=$(awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' cert.crt)`
But replacing line in the .env file I can't do...